authors/apps/authentication/tests/test_views.py

```diff
@@ -44,3 +46,151 @@ def test_edit_user_credentials(self):
         username = user_update['user']['username']
         self.assertEqual(response.data['username'], username)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class TestResetPassword(APITestCase):
+    def setUp(self):
```

Comment:
@moseskamira Could you sync with @fahadmak and use some of the data in `data.py` or any data you want there tat could be missing.